publish.py

```diff
@@ -21,9 +21,7 @@
 
 shutil.rmtree("build", ignore_errors=True)
 
-subprocess.run(shlex.split("python3 -m build --sdist"), check=True)
 subprocess.run(shlex.split("python3 -m build --wheel"), check=True)
-sdist = f"dist/{PKG_NAME}-{version}.tar.gz"
```

Comment:
Does this script run now? Sdist is still in the list below, so I’d expect it to throw. I think we should publish sdist, so I’d revert this line change and keep it in the twine iterator.

we should probably also rm all files in ‘dist’ similar to how we clear out ‘build’ too, right? If we are publishing from CI only, then this doesn’t matter, but if we are publishing from desktop then it matters a lot.

let’s make sure top level build and dist are in gitignore (I think they are) and do a full clean of those folders via this script before publishing.

Reply:
Good catch!  I'll remove sdist from the list.  I probably keep the loop and revisit adding sdist later.  I'll push a commit in a bit...